File: pyg_parent/interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.entity.SeckillEntity;
import cn.itcast.core.pojo.seckill.SeckillGoods;

public interface SeckillService {

    /*商家申请秒杀商品*/
    void add(SeckillGoods seckillGoods);

    PageResult search(Integer page, Integer rows, SeckillGoods seckillGoods);

    void updateStatus(Long[] ids, String status);
}

File: pyg_parent/web_manager/src/main/webapp/js/service/seckillService.js
app.service("seckillService",function($http){
    this.search = function(page,rows,searchEntity){
        return $http.post("../seckill/search.do?page="+page+"&rows="+rows,searchEntity);
    }

    this.seckillUpdateStatus = function(ids,status){
        return $http.get('../seckill/updateStatus.do?ids='+ids+"&status="+status);
    }
});